3-extra/1-choose-your-own-adventure.js

```diff
@@ -44,6 +44,10 @@ let game = {
     // object for the correct room.
     //
     // Hint: the only valid rooms are "hall", "classroom" and "library".
+
+    if (roomName === ("hall" || "classroom" || "library")) {
```

Comment:
yes, or `['hall', 'classroom', 'library'].includes(roomName)`

Reply:
Actually this does not work. `roomName === ("hall" || "classroom" || "library")` is just `roomName === "hall"`; `||` returns the first truth-y value (see [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR)), the other two values are ignored entirely

Reply:
Oh, You're right and I didn't get error because I always tried the game starting with the hall. :)
Just fixed it. Thanks!